website/docs/developer/research/research-graphql.md

```diff
@@ -142,48 +142,46 @@ In case of using Python library like [Graphene](https://github.com/graphql-pytho
   
   
   A few suggestions are given below:
-```commandline
-#---------------------------post_copy_request_to_queue----------------------
 
-  ## developers might need to modify get_metadata_sql(key_path) and use the graphql query. See https://docs.graphene-python.org/en/latest/execution/execute/.
-  ## update test_post_copy_request_to_queue.py based on changes in post_copy_request_to_queue.py. One test could be `test_get_metadata_sql_happy_path()`
-  ## shared_recovery.update_status_for_file(), shared_recovery.post_entry_to_queue() function for sending to SQS might need to be removed and replace this with code for writing to the DB.
-  ## Additional changes are expected.
+#### post_copy_request_to_queue
 
-# --------------------copy_files_to_archive--------------------------------
+ - developers might need to modify `get_metadata_sql(key_path)` and use the graphql query. See https://docs.graphene-python.org/en/latest/execution/execute/.
+ - update `test_post_copy_request_to_queue.py` based on changes in `post_copy_request_to_queue.py`. One test could be `test_get_metadata_sql_happy_path()`
+ - `shared_recovery.update_status_for_file()`, `shared_recovery.post_entry_to_queue()` function for sending to SQS might need to be removed and  replaced with code for writing to the DB.
+ - Additional changes are expected.
 
-  ## shared_recovery.update_status_for_file(), shared_recovery.post_entry_to_queue() functions for sending to SQS might need to be removed and replaced with code for writing to the DB.
-  ## Additional changes are expected.
+#### copy_files_to_archive
 
+ - `shared_recovery.update_status_for_file()`, `shared_recovery.post_entry_to_queue()` functions for sending to SQS might need to be removed and replaced with code for writing to the DB.
```

Comment:
Change `,` to `and` and potentially change `for writing to the DB.` to something that references GraphQL. Maybe `that leverages GraphQL to write to the database.`